airflow/providers/daskexecutor/executors/dask_executor.py

```diff
@@ -135,13 +135,13 @@ def end(self) -> None:
             assert self.client
             assert self.futures
 
-        self.client.cancel(list(self.futures.keys()))
+        self.client.cancel(list(self.futures))
         for future in as_completed(self.futures.copy()):
             self._process_future(future)
 
     def terminate(self):
         if TYPE_CHECKING:
             assert self.futures
 
-        self.client.cancel(self.futures.keys())
+        self.client.cancel(list(self.futures))
```

Comment:
Hey this might be one of those cases we should use `keys()` instead. (Or can we just pass in the dict?)